docs/modules/administration/pages/installation.adoc

```diff
@@ -159,8 +162,10 @@ make install-sonarqube
 ----
 
 This will launch an instance of SonarQube.
-The script will prompt for a new admin password, set it and create an auth token to be used by the Jenkins pipelines.
-Both values can be written into `ods-configuration/ods-core.env`, which you then need to commit and push before continuing.
+The script will prompt for a new admin password, once set, an auth token that is used by the Jenkins pipelines will be generated.
+Both values are automatically written into `ods-configuration/ods-core.env`, which you then need to commit and push after adding the Bitbucket `ods-configuration` repo as the remote origin.
```

Comment:
Isn't the user asked earlier in the guide to push this repo already? Then the remote should be set already.

Reply:
moved it to the bitbucket part!